src/loki/search.cc

```diff
@@ -379,21 +379,23 @@ struct bin_handler_t {
       if(!candidate.edge->forward())
         length_ratio = 1.f - length_ratio;
       //side of street
-      auto side = candidate.get_side(location.latlng_, score);
+      auto side = candidate.get_side(location.latlng_, candidate.sq_distance);
+      PathLocation::PathEdge path_edge{candidate.edge_id, length_ratio, candidate.point, score, side, get_reach(candidate.edge)};
```

Comment:
i liked the way the node correlation made the edges ahead of the `if`s and then `std::move`d them into the `vector`s so i followed suite here